LovelyCrane/Manager/KeyboardObserver.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  KeyboardObserver.swift
+//  LovelyCrane
+//
+//  Created by 황지우2 on 2023/07/19.
+//
+
+import SwiftUI
+
+class KeyboardObserver: ObservableObject {
```

Comment:
KeyboardObserver가 상속되는 일이 있을까요 ? 
없다면 final 설정을 하면 좋을거 같습니당 ㅎㅎㅎㅎ
```suggestion
final class KeyboardObserver: ObservableObject {
```

Reply:
감사합니다! 그렇게 변경해볼게요